cmake/EthOptions.cmake

```diff
@@ -43,6 +43,21 @@ macro(configure_project)
 		add_definitions(-DETH_JSCONSOLE)
 	endif()
 
+	# CI Builds should provide (for user builds this is totally optional)
+	# -DBUILD_NUMBER - A number to identify the current build with. Becomes TWEAK component of project version.
+	# -DVERSION_SUFFIX - A string to append to the end of the version string where applicable.
+	if (NOT DEFINED BUILD_NUMBER)
+		# default is very big so that local build is always considered greater
+		# and can easily replace CI build for for all platforms if needed
+		set(BUILD_NUMBER 9999999)
```

Comment:
So only 2 bytes per version component. Still should be enough. Let me change it.